### DISCUSSION TITLE
为什么官网都不更新了呢？

### DISCUSSION THREAD

**Discussion by ningmonguo** [eolinker/apinto#159] [General]
我看官网的教程现在还是基于 v0.4.0 的 https://www.gokuapi.com/
程序现在都迭代到 17 了

**Comment by Dot-Liu** [comment]
官网是：https://www.apinto.com
控制台部分教程网站：https://help.eolink.com/tutorial/Apinto/c-1406
节点部分教程网站：https://help.apinto.com/docs

**Reply by ningmonguo** [reply]
非常感谢，原来项目已经合并了。
再请问下 现在的apinto怎么读呢？

**Comment by Dot-Liu** [comment]
核心代码都在eosc这个仓库里
![image](https://github.com/eolinker/apinto/assets/14105999/107ab953-f0d5-4b1b-99d0-df257318cbb3)

入口在apinto里
![image](https://github.com/eolinker/apinto/assets/14105999/0b554687-251e-4411-bdd5-1158bbb890ed)
